language/en/extimgaslink.php

```diff
@@ -23,5 +23,11 @@
 }
 
 $lang = array_merge($lang, array(
+	'EXTIMGASLINK_CONFIG'			=> 'Allowed origin from images used in posts',
+	'EXTIMGASLINK_CONFIG_EXPLAIN'	=> 'Other images are replaced with links.',
+	'EXTIMGASLINK_INVALID_CONFIG'	=> 'The origin you specified is not valid.',
+	'EXTIMGASLINK_SECURE_SITES'		=> 'only images from secure websites and the server',
+	'EXTIMGASLINK_SERVER_ONLY'		=> 'only images from the server',
```

Comment:
Hier würde ich zusätzlich "Not valid " durch "invalid " ersetzen, außerdem "All other images " statt "other images "